acceptance/ws_test/recipes/default.rb

```diff
@@ -0,0 +1,9 @@
+#
+# Cookbook:: ws_test
+# Recipe:: default
+#
+
+chef_ingredient "chef-workstation" do
```

Comment:
The channel defaults to stable - we could set current but the catch here is that it can only tests released builds (unstable, current, stable) so if we wanted to test each we could create a suite in kitchen with an attribute.

Reply:
Would it be possible to install via artifact from earlier in the build? Perhaps something like the now-defunct readme updater, which updated the download URL.

Reply:
Depending on how we integrate this into our build pipeline I think we should be able to test every merged PR. Maybe that means installing from the `unstable` channel? Or using the output of a previous build to install from a local artifact?

Either way, I think we could get the kitchen framework setup in this PR and then improve it in another to have it run from the equivalent of `unstable` instead of `stable`